src/prompt.c

```diff
@@ -446,6 +446,7 @@ readline_init(void)
 
 static void sigint_absorb_handler(int sig) {
 	signal(SIGINT, SIG_DFL);
+	redraw_display(true);
```

Comment:
It feels a bit unsafe to call this inside the signal handler. Mind changing it to use a flag?